src/header.js

```diff
@@ -1,67 +1,28 @@
 import React, { Component } from 'react';
 import Scrollspy from 'react-scrollspy';
-
+import HeaderButton from './headerButton.js';
 import './style/header.css';
 
 class Header extends Component {
-
-  // componentDidMount() {
-  //     var s1 = document.getElementById('s1');
-
-  //     s1.on('click', function(event) {
-
-  //     });
-  // }
-
   render() {
 
-
-
-
-
-/*** codigo original exemplo.  https://stackoverflow.com/questions/17631417/css-pure-css-scroll-animation/17633941#17633941
-
-    $(document).ready(function(){
-  // Add smooth scrolling to all links
-  $("a").on('click', function(event) {
-
-    // Make sure this.hash has a value before overriding default behavior
-    if (this.hash !== "") {
-      // Prevent default anchor click behavior
-      event.preventDefault();
-
-      // Store hash
-      var hash = this.hash;
-
-      // Using jQuery's animate() method to add smooth page scroll
-      // The optional number (800) specifies the number of milliseconds it takes to scroll to the specified area
-      $('html, body').animate({
-        scrollTop: $(hash).offset().top
-      }, 800, function(){
-   
-        // Add hash (#) to URL when done scrolling (default click behavior)
-        window.location.hash = hash;
-      });
-    } // End if
-  });
-});
-****/
     return (
       <ul className="header">
           <li className='logo'>
             <img src="http://img1.topimagens.com/ti/img/facebook_42x42.png"/>
           </li>
           <li className='menu'>
-            <Scrollspy items={ ['headerAnchor', 'massotherapyAnchor', 'section-3'] } 
+            <Scrollspy items={ ['header-anchor', 'massotherapy-anchor', 'section-3'] } 
               currentClassName="is-current">
-              <li className="s1" ><a href="#header-anchor">section 1</a></li>
-              <li className="s2" ><a href="#massotherapy-anchor">section 2</a></li>
-              <li className="s3" ><a href="#section-3">section 3</a></li>
+              <HeaderButton id={"s1"} title={"Section 1"} anchoredTo={"header-anchor"} />
+              <HeaderButton id={"s2"} title={"Section 2"} anchoredTo={"section-3"}/>
+              <HeaderButton id={"s3"} title={"Section 3"} anchoredTo={"massotherapy-anchor"} />
+              <HeaderButton id={"s4"} title={"Section 4"} anchoredTo={"massotherapy-anchor"} />
             </Scrollspy>
           </li>
       </ul> 
     );
   }
 }
 
-export default Header;
+export default Header;
```

Comment:
I would suggest to add a newline character to the end of the file to avoid problems like these:

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

But it is up to you :)

Reply:
That's right! Fair enough. A newline were added to all the files you noticed it missing